Vue InstantSearch/query-suggestions/src/App.vue

```diff
@@ -95,20 +96,25 @@ export default {
       }
     },
     indicesToSuggestions(indices) {
-      return indices.map(({ hits }) => ({
-        data: hits.map(hit => {
-          const [category] =
-            (hit.instant_search &&
-              hit.instant_search.facets.exact_matches.categories) ||
-            [];
-
-          return {
-            ...hit,
-            category,
-            name: hit.query,
-          };
-        }),
-      }));
+      return indices
+        .filter(
+          ({ indexName }) =>
+            indexName === 'instant_search_demo_query_suggestions'
```

Comment:
since the other index is now adjacent, you also get access to its results in the autocomplete index.